### ISSUE TITLE
Make Layer Properties Dialog Draggable 

### ISSUE THREAD

**Issue by gitspeaks** [PintaProject/Pinta#912]
Currently, the Layer Properties dialog, which appears when double-clicking a layer to adjust transparency, is fixed in the center of the screen. This placement often obstructs important visual areas, hindering the ability to effectively preview changes live as transparency levels are adjusted.

Allowing the dialog to be moved freely across the screen would prevent it from covering crucial areas, thereby improving visibility and interaction during adjustments.

**Comment by cameronwhite** [comment]
Which platform are you on? This might be something specific to the window manager's behaviour - testing on macOS the dialog can freely move

**Comment by gitspeaks** [comment]
Standard Ubuntu 24.04 LTS desktop installation.

**Comment by cameronwhite** [comment]
I think this might be a behaviour of GNOME shell - e.g. https://popey.com/blog/2021/01/my-gnome-tweaks/ mentions that the GNOME tweaks app can allow you to change whether modal dialogs are immovable or not
Let me know if that works for you?

I think the dialog *should* be modal with Pinta's current behaviour, since allowing interaction with the main window probably won't work without a lot more code changes to deal with the layer being removed while the dialog is open, etc

**Comment by gitspeaks** [comment]
I personaly avoid installing "tweak" tools, don't know if a tweak will come and bite me in a different use-case... 

In any case, the issue is not about allowing  interaction with the "main" window but  allowing to move the modal (like on the mac) so you can interact with it when placed in an area that does not obstructs important information, for example, moving it aside to see a live preview of how different transparency settings affect info at the center of the screen.

**Comment by cameronwhite** [comment]
Right, but I don't think Pinta specifically is blocking the dialog from moving , but rather that's the default behaviour for these types of dialogs on your OS?

**Comment by gitspeaks** [comment]
Correct. Apparently, this is a [well known](https://www.reddit.com/r/gnome/comments/1dbzni6/dear_gnome_devs_attach_modal_dialogs_should_be/)  behavior in GNOME. I wouldn't hold my breath for it to get resolved. I think seeing a live preview of different opacity settings is quite useful. Please consider allowing customization of the UI to show the slider in the horizontal toolbar in the main window to avoid dealing with the fixed dialog issue.

**Comment by cameronwhite** [comment]
Another alternative might be changing the dialog into a non-modal dialog or a `Gtk.Popover`, along with handling any potential issues with it no longer being model (e.g. the layer being removed while the dialog is open)
